File: maven/maven-library/src/main/java/dev/frontseat/library/FancyGreeter.java
package dev.frontseat.library;

import dev.frontseat.common.Greeter;

public class FancyGreeter {

    public static final String VERSION = "0.2.0";

    private final Greeter greeter = new Greeter();

    public String greet(String name) {
        return "*** " + greeter.greet(name) + " ***";
    }
}

File: dotnet/dotnet-library/FancyGreeter.cs
using CommonDotnetLibrary;

namespace DotnetLibrary;

public static class FancyGreeter
{
    public const string Version = "0.2.0";

    public static string Greet(string name)
    {
        return $"*** {Greeter.Greet(name)} ***";
    }
}
